components/ViewBills/ViewBills.js

```diff
@@ -84,6 +85,7 @@ const ViewBills = (props) => {
           <option value="id">Bill #</option>
           <option value="cosponsorCount"># CoSponsors</option>
           <option value="testimonyCount"># Testimony</option>
+          <option value="hearingDate">Next Hearing Date</option>
```

Comment:
Sometimes this field is blank because a hearing has not been scheduled.  It might be clearer if this column heading was "Hearing Scheduled".